code/modules/experisci/experiment/types/explosive.dm

```diff
@@ -0,0 +1,39 @@
+/datum/experiment/explosion
+	name = "Explosive Experiment"
+	description = "An experiment requiring an explosion to progress"
+	exp_tag = "Explosion"
+	performance_hint = "Perform explosive experiments using the research doppler array in the toxins lab."
+	/// The required devastation range to complete the experiment
+	var/required_devastation = 0
+	/// The required heavy impact range to complete the experiment
+	var/required_heavy = 0
+	/// The required light impact range to complete the experiment
+	var/required_light = 0
+	/// The last measured devastation range
+	var/last_devastation
+	/// The last measured heavy range
+	var/last_heavy
+	/// The last measured light range
+	var/last_light
+
+/datum/experiment/explosion/is_complete()
+	return required_devastation <= last_devastation \
+		&& required_heavy <= last_heavy \
```

Comment:
```suggestion
	return required_devastation <= last_devastation
		&& required_heavy <= last_heavy
```

Feel free to tell me if this doesn't work, but I'm pretty sure backslashes are only necessary in macros.

Reply:
this doesnt work